openstates/ny/bills.py

```diff
@@ -64,10 +67,24 @@ def _parse_bill_details(self, bill):
             "http://www.nysenate.gov/legislation/bills/{bill_session}/" "{bill_id}"
         ).format(bill_session=bill["session"], bill_id=bill_id)
 
+        # assembly_url = (
+        #     "http://assembly.state.ny.us/leg/?default_fld=&bn={bill_id}"
+        #     "&Summary=Y&Actions=Y&Text=Y"
+        # ).format(bill_id=bill_id)
+        assembly_bill_id = bill_id
+        if (bill_id[-1] == "A") or (bill_id[-1] == "B"):
+            assembly_bill_id = bill_id[:-1]
+        if len(assembly_bill_id) == 3:
```

Comment:
might be better to use string formatting on this

something like `"{}{:04d}".format(prefix, num)`